pycaching/cache.py

```diff
@@ -674,14 +684,91 @@ def load_quick(self):
 
         logging.debug("Cache loaded: {}".format(self))
 
+    def load_by_guid(self):
+        """Load cache details using the GUID to request and parse the caches
+        'print-page'. Loading as many properties as possible except the
+        following ones, since they are not present on the 'print-page':
+
+          + original_location
+          + state
+          + found
+          + pm_only
+
+        :raise .PMOnlyException: If the PM only warning is shown on the page
+        """
+        if not hasattr(self, "guid"):
+            self._load_guid()
+        res = self.geocaching._request(self._urls["print_page"],
+                                       params={"guid": self.guid})
+        if res.find("p", {"class": "Warning"}) is not None:
+            raise errors.PMOnlyException()
+        content = res.find(id="Content")
+
+        self.name = content.find("h2").get_text()
+
+        self.location = Point.from_string(
+            content.find("p", {"class": "LatLong Meta"}).text)
+
+        type_img = os.path.basename(content.find("h2").img.get("src"))
```

Comment:
Actually the _src_ attribute of `img` tags seems to be always accessed via `get("src")` all over the code. Do you want me to change them all (even outside `load_by_guid`)?